tests/cluster/tests/21-many-slot-migration.tcl

```diff
@@ -1,12 +1,13 @@
 # Tests for many simlutaneous migrations.
 
+# TODO: Test is currently disabled until it is stabilized (fixing the test
+# itself or real issues in Redis).
+
+if {false} {
+
 source "../tests/includes/init-tests.tcl"
 source "../tests/includes/utils.tcl"
 
-# TODO: This test currently runs without replicas, as failovers (which may
-# happen on lower-end CI platforms) are still not handled properly by the
-# cluster during slot migration (related to #6339).
-
```

Comment:
are you sure you wanna remove this comment?
you didn't revert the change you introduced with this comment, so i suppose it's still valid.
re-enabling the test one day, we may also want to re-enable replicas too.